doc/usage.rst

```diff
@@ -54,6 +54,20 @@ However, the tool makes a few assumptions of your project's structure:
 After these are place, you can just build the package with your
 favorite tool!
 
+Environmental variables
```

Comment:
It's "environment variables", not "environmental". The first one has to do with computers, the second with the physical world and the climate.